client/package.json

```diff
@@ -430,6 +430,18 @@
         "title": "BitBake: Parse all recipes",
         "description": "This command runs bitbake in parse-only mode."
       },
+      {
+        "command": "bitbake.start-toaster-in-browser",
+        "title": "BitBake: Start toaster in browser",
+        "description": "Start the toaster and reveal it in the browser.",
+        "icon": "$(preview)"
+      },
+      {
+        "command": "bitbake.stop-toaster",
+        "title": "BitBake: Stop Toaster",
+        "description": "This stops the toaster by running `source toaster stop`.",
```

Comment:
no need to prefix toaster with "the". It's the tool's name

Reply:
done.